### PR TITLE
Update stale ioctl

### PR DESCRIPTION
**Pull request by koseoyoung**
1. change name auto-restake -> auto-stake
2. delete next flag in ```node delegate```
3. fix typo 
4. delete action deploy/invoke commands

### CODE REVIEW

File: ioctl/cmd/action/stake2.go [hunk 1]
```diff
@@ -34,9 +34,13 @@ var (
 		config.English: "insecure connection for once (default false)",
 		config.Chinese: "一次不安全的连接（默认为false)",
 	}
+	stake2FlagAutoStakeUsages = map[config.Language]string{
+		config.English: "auto stake boost: the voting power will not decrease until disable it",
```
Comment (dustinxie) [inline ioctl/cmd/action/stake2.go:38]: config.English: "auto-stake boost: the voting power will not decrease"

File: ioctl/cmd/action/stake2.go [hunk 2]
```diff
@@ -34,9 +34,13 @@ var (
...
+		config.Chinese: "",
+	}
```
Comment (dustinxie) [inline ioctl/cmd/action/stake2.go:40]: "自动质押，权重不会衰减"

File: ioctl/cmd/action/actiondeploy.go [hunk 3]
```diff
@@ -1,63 +0,0 @@
-// Copyright (c) 2019 IoTeX Foundation
```
Comment (dustinxie) [inline ioctl/cmd/action/actiondeploy.go:1]: contract deploy/invoke cmd are still WIP, for now keep these 2 files
we can revisit and make the decision to delete or not, once the contract cmds are done

File: ioctl/cmd/action/actioninvoke.go [hunk 4]
```diff
@@ -1,67 +0,0 @@
-// Copyright (c) 2019 IoTeX Foundation
```
Comment (dustinxie) [inline ioctl/cmd/action/actioninvoke.go:1]: same as above

File: ioctl/cmd/action/actiondeploy.go (hunk 3 above)
Comment (koseoyoung) [reply ioctl/cmd/action/actiondeploy.go:1]: Ok, I will make an issue for it then

**Pull request merged**